README.md

```diff
@@ -52,6 +52,9 @@ make sure that you do before you proceed.**
 * [Gradle (2.12+)](https://gradle.org/)
 * [Git](https://git-scm.com/download/)
 
+## Installation Guides
+[Windows](documentation/Windows-Installation-Guide.md)
```

Comment:
Here I would suggest to add also macOS and Debian based (or only Ubuntu) even if we don't have the link to them yet. Saying something like (coming soon ...). To say both that we are working on them and that it works on them too.